### PR TITLE
add integration test

### LINKED ISSUE #12: test rabbitmq cluster in the integration test
##### ISSUE TYPE

 - Bug Report

##### ROLE VERSION

unreleased

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
N/A

##### SUMMARY

the current integration test does not intensively test the role.

##### STEPS TO REPRODUCE
N/A

##### EXPECTED RESULTS

the role should be thoroughly tested in the integration test.

##### ACTUAL RESULTS
the test is nothing more than starting and connecting rabbitmq.

### PR DESCRIPTION
**Pull request by trombik**
fixes #12 

see also #14 and #15

### CODE REVIEW

File: tasks/plugins.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@
   when: rabbitmq_plugins_disable | length > 0
 
 - name: Get list of enabled plugins
-  shell: "rabbitmq-plugins -E list | grep -E '^\\[' | cut -f 2 -d ' '"
+  shell: "rabbitmq-plugins list -E | grep -E '^\\[' | cut -f 2 -d ' '"
```
Comment (mitsururike) [inline tasks/plugins.yml]: `rabbitmq-plugins list -m -E` should be used.

File: tests/serverspec/default_spec.rb [hunk 2]
```diff
@@ -112,8 +117,30 @@
...
-describe command("rabbitmq-plugins list") do
+describe command("rabbitmq-plugins list -E -m") do
   its(:exit_status) { should eq 0 }
   its(:stderr) { should eq "" }
-  its(:stdout) { should match(/\s+rabbitmq_management\s+\d+\.\d+\.\d+/) }
+  its(:stdout) { should match(/^rabbitmq_management$/) }
+  its(:stdout) { should_not match(/^rabbitmq_trust_store$/) }
+end
+
+describe command("rabbitmqctl list_users | grep -v '^Listing users'") do
```
Comment (mitsururike) [inline tests/serverspec/default_spec.rb]: `rabbitmqctl list_users -q` should be used.

File: tasks/plugins.yml [hunk 3]
```diff
@@ -56,7 +56,7 @@
   when: rabbitmq_plugins_disable | length > 0
 
 - name: Get list of enabled plugins
-  shell: "rabbitmq-plugins -E list | grep -E '^\\[' | cut -f 2 -d ' '"
+  shell: "rabbitmq-plugins list -E -m"
```
Comment (mitsururike) [inline tasks/plugins.yml]: `command` should be used instead of `shell`.

File: tasks/configure-cluster.yml [hunk 4]
```diff
@@ -0,0 +1,58 @@
+---
+
+- assert:
+    that:
+      - rabbitmq_cluster_name is defined
+      - rabbitmq_cluster_name | length > 0
+
+- name: Get cluster name
+  shell: "rabbitmqctl eval 'io:fwrite(\"~s~n\", [rabbit_nodes:cluster_name()]).' | grep -v '^ok$'"
```
Comment (mitsururike) [inline tasks/configure-cluster.yml]: `sed '$d'` should be used to remove 'ok' from the erlang shell because the cluster name may be 'ok'.

Comment (trombik) [inline tasks/configure-cluster.yml]: i prefer `head(1)`

**Pull request merged**